lib/containers/common.pm

```diff
@@ -43,7 +44,7 @@ sub install_podman_when_needed {
             assert_script_run "apt-get -y install podman", timeout => 220;
         }
         else {
-            add_suseconnect_product('sle-module-containers') if (is_sle '>=15');
+            add_suseconnect_product('sle-module-containers') if ($is_sle_host && is_sle '>=15');
```

Comment:
Is `is_sle` true for tests on OSD using openSUSE HDDs? If so, that's IMO fundamentally broken

Reply:
the answer is yes. We still use sle as DISTRI. the reason is because the SUT images are sle. This cause some confusion but this is how we approached it with other distributions and we agreed to keep the same for opensuse

Reply:
Wait, I agree with @b10n1k but what if we test SLE12-sp5 based image on openSUSE?

Reply:
i dont know. this caused all the confusion. there is no such scenario as for now and if we have based image on openSUSE on o3 ever, this job would not make sense. the intention is to not run the based images there, right? i am resolving this. feel free to reopen.

Reply:
This is actually OK. Agree

Reply:
That is indeed very confusing, but it seems like only one place needs a hack, which is tolerable.